doc/Language/exceptions.pod6

```diff
@@ -81,9 +86,18 @@ almost the same information as the normal backtrace printer.
          }
     }
 
-Note that the match target is a role. To allow user defined exceptions to match
-in the same manner, they must implement the given role. Just existing in the
-same namespace will look alike but won't match in a C<CATCH> block.
+While this is a very common pattern, it is not strictly necessary to use
+C<default> or C<when> in a C<CATCH> block. This is done to prevent the
+control flow in one from reaching its end where, unless the exception
+has been L<resumed|#Resuming_of_exceptions>, the exception will continue
+to be thrown.  Allowing this can be used for logging purposes, for
+instance:
```

Comment:
up to where? The end of the catch block?

Reply:
Yes. "the end of the block" might be clearer than "its end" here.